airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/spec.json

```diff
@@ -16,6 +19,15 @@
         "description": "Api key of bamboo hr",
         "airbyte_secret": true
       },
+      "start_date": {
```

Comment:
Adding a new parameter is a breaking change to existing connections.

Reply:
Yes but we need this parameter to do incremental